doc/guides/tfm/testsuites.rst

```diff
@@ -0,0 +1,63 @@
+Test Suites
+###########
+
+TF-M includes two sets of test suites:
+
+* tf-m-tests - Standard TF-M specific regression tests
+* psa-arch-tests - Test suites for specific PSA APIs (secure storage, etc.)
+
+These test suites can be run from Zephyr via an appropriate sample application
+in the samples/tfm_integration folder.
+
+TF-M Regression Tests
+*********************
+
+The regression test suite can be run via the ``tfm_regression_test`` sample,
+for example:
+
+   .. code-block:: bash
+
+     $ west build samples/tfm_integration/tfm_regression_test/ \
```

Comment:
I don't think I agree 100% with avoiding any sort of repetition on the documentation is always positive, but I've removed all build commands and link to the samples now since two of you pointed out the same concern.